doc/source/conf.py

```diff
@@ -306,11 +311,17 @@
         "json_url": f"https://{cname}/versions.json",
         "version_match": switcher_version,
     },
-    "use_meilisearch": {
-        "api_key": os.getenv("MEILISEARCH_PUBLIC_API_KEY", ""),
-        "index_uids": {
-            f"pymapdl-v{switcher_version.replace('.', '-')}": "PyMAPDL",
-        },
+    # "use_meilisearch": {
+    #     "api_key": os.getenv("MEILISEARCH_PUBLIC_API_KEY", ""),
+    #     "index_uids": {
+    #         f"pymapdl-v{switcher_version.replace('.', '-')}": "PyMAPDL",
+    #     },
+    # },
+    "cheatsheet": {
+        "file": "cheat_sheet/cheat_sheet.qmd",
+        "title": "PyMAPDL cheat sheet",
+        "version": f"v{version}",
+        "pages": ["index", "getting_started/learning"],
```

Comment:
What is this for?

Reply:
This is the configuration needed for the cheat sheet build process. It's part of the theme's options.